ui/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "webpack-dev-server --config webpack.dev.js",
     "build": "webpack --config webpack.prod.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest"
```

Comment:
If you'd like to use the `_test.js` suffix instead of `.test.js`, it looks like you can add [configuration](https://jestjs.io/docs/en/configuration#testmatch-array-string), either as a separate file, or directly in `package.json`:

```json
{
  "jest": {
    "testMatch": [...],
  },
  "scripts": {
    "test": "jest",
  },
  "devDependencies": {
    "jest": "^24.1.0",
  }
}
```

Reply:
Cool, done.